CONTRIBUTING.md

```diff
@@ -110,6 +112,17 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
 If you set your `user.name` and `user.email` git configs, you can sign your
 commit automatically with `git commit -s`.
 
+## Integration Tests
```

Comment:
This seems interesting mostly to PR submitters, in which case we want this header one level deeper (`##` -> `###`) and can skip the table-of-contents entry.

Reply:
Good point, doing this stays more in line with the Sign your PR's section.  Changed